packages/browser/src/client/runner.ts

```diff
@@ -39,8 +42,8 @@ export function createBrowserRunner(original: any, coverageModule: CoverageHandl
       }
     }
 
-    async onAfterRunFiles() {
-      await super.onAfterRun?.()
+    async onAfterRunFiles(files: File[]) {
+      await super.onAfterRunFiles?.(files)
```

Comment:
I remember there being some changes done here earlier:

- https://github.com/vitest-dev/vitest/issues/3514
- https://github.com/vitest-dev/vitest/pull/3806

I'll need to re-check this later today.

Reply:
Looks good!